site/content/en/docs/formats/super_resolution.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Super Resolution'
+linkTitle: 'Super Resolution'
+description: ''
+weight: 2
+---
+
+## Format specification
+
+Super resolution format specification is available [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/README.md#supported-converters).
+
+Supported attributes:
+- `HR` (`Image`): high resolution image
+- `upsampled` (`Image`): upsampled image
```

Comment:
From the description I see that `upsampled` is optional, but `HR` is not. Also, `HR` is in lower case in the implementation.

From the format semantics viewpoint, it would be better to represent these attributes as annotations. Attributes can work too, but I think it will be changed later, because they do not seem to be compatible with other formats.

Reply:
Do you suggest adding a new annotation type for this?

Reply:
Ideally, I'd prefer this solution.

Reply:
Added `ImageResolution` annotation type

Reply:
Added `ImageResolution` annotation type